keyboards/nimrod/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUTS = ortho_4x10
```

Comment:
@fauxpark I've added this line. Just curious -- Why do I need to list only this one layout? Should I list all five?

Reply:
This is for the [Community Layouts](https://docs.qmk.fm/#/feature_layouts) feature. Basically, instead of defining the same keymap for every board you have with the same layout, you can add it to the `layouts/` directory instead, and you'll be able to compile it for any board that supports it. Only the grid (ortho_4x10) layout is applicable here, as the others are not as common.

Reply:
Ah whoops. Thought I committed that suggestion, but I guess I just marked it as resolved. Sorry about that. We should be all good now.

Reply:
You did.